### PR TITLE
Moving things to more appropriate locations

### PR DESCRIPTION
**Pull request by jmthibault79**
Attempt to move a few things to be more in line with apparent standards.

I was confused by this for a bit while working on the other PR.

Please feel free to ignore or close.

I, the developer opening this PR, do solemnly pinky swear that:

- [x] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [x] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [x] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala [hunk 1]
```diff
@@ -57,8 +57,8 @@ trait FireCloudApiService extends CookieAuthedApiService
   with WorkspaceApiService
   with NotificationsApiService
   with MethodConfigurationApiService
-  with BillingService
-  with SubmissionService
+  with BillingApiService
```
Comment (jmthibault79) [inline src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala:60]: The standard for API route handlers seems to be:
* named ...ApiService
* lives in the webservice package

So I changed these 3 to fit

File: src/test/scala/org/broadinstitute/dsde/firecloud/webservice/ApiServiceSpec.scala [hunk 2]
```diff
@@ -1,16 +1,12 @@
-package org.broadinstitute.dsde.firecloud.service
```
Comment (jmthibault79) [inline src/test/scala/org/broadinstitute/dsde/firecloud/webservice/ApiServiceSpec.scala:1]: Several spec files were in the apparently-wrong package too

**Review by MatthewBemis — APPROVED** [review]
:+1: for everything in here, it's all in line with what we're trying to do in terms of re-organizing the code. I had started to do a chunk of this re-org in the [akka-http migration](https://github.com/broadinstitute/firecloud-orchestration/pull/802) but some of it was out of scope so I left it for another time. I'm sorry that you had to deal with it in the interim but I am super grateful that you're leaving things in better shape than they were when you found them! 🏅

**Comment by jmthibault79** [comment]
jenkins retest

**Pull request merged**